fbw-a380x/src/localization/msfs/de-DE.locPak

```diff
@@ -15,8 +15,8 @@
       "AIRCRAFT.UI_MANUFACTURER": "Airbus",
       "AIRCRAFT.UI_MODEL": "A380",
       "AIRCRAFT.UI_VARIATION": "FlyByWire A380X (A380-842)",
-      "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Ausgewählten Steuerkurs nach links anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
-      "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "Ausgewählten Steuerkurs nach rechts anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)"
+      "A380.COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Ausgewählten Steuerkurs nach links anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
```

Comment:
You only have to do the en-US.locPak - the rest is handled on Localazy. It downloads the latest language files every build.
So if you add a key to the en-US.locPak we will upload it to Localazy and it will automatically be added to all languages. The translation then also happens on Localazy.